pkg/psmdb/backup/pbm.go

```diff
@@ -153,7 +153,11 @@ func secret(cl client.Client, namespace, secretName string) (*corev1.Secret, err
 type LockHeaderPredicate func(pbm.LockHeader) bool
 
 func NotPITRLock(l pbm.LockHeader) bool {
-	return l.Type != pbm.CmdPITR
+	return !IsPITRLock(l)
```

Comment:
Not friendly at all =)
Function which returns inverted result of another function.
It's worth to return it back, or invert result on `caller` side

Reply:
Changed it back in 6c687095c2f1f239b153f309c24a2a90914ea436